ruleset.go

```diff
@@ -123,19 +80,25 @@ func (r *Ruleset) validate() error {
 
 // Signature represents the signature of a ruleset.
 type Signature struct {
-	ReturnType string
-	ParamTypes map[string]string
+	ReturnType string            `json:"returnType"`
+	Params     map[string]string `json:"params"` // TODO(asdine) rename to Params
```

Comment:
The TODO troubles me - rename what to Params?

Reply:
I forgot to remove the TODO :laughing: